src/engine/input/key_mapping/InputGroupList.cs

```diff
@@ -131,17 +131,17 @@ public InputEventItem Conflicts(InputEventItem item)
         latestDialogConflict = conflict;
         latestDialogNewEvent = newEvent;
 
-        conflictDialog.DialogText = string.Format(CultureInfo.CurrentCulture,
+        conflictDialog.GetNode<Label>("Label2").Text = string.Format(CultureInfo.CurrentCulture,
```

Comment:
couldn't you give Label2 a better name 😄

Reply:
Changed to `DialogText` :)